novawallet/Common/View/ScrollableContainerView/ScrollableContainerView.swift

```diff
@@ -22,6 +22,12 @@ final class ScrollableContainerView: UIView {
         }
     }
 
+    var scrollContentBottomOffset: CGFloat = 0 {
```

Comment:
We usually use stackView.layoutMargins to setup the insets directly in the layout